RSEF announcement

tacaswell opened the pull request.

tacaswell committed:
RSEF announcement

QuLogic commented on content/posts/matplotlib-rsef/index.md:
```diff
@@ -0,0 +1,51 @@
+
+As has been discussed in
+[detail](https://www.fordfoundation.org/work/learning/research-reports/roads-and-bridges-the-unseen-labor-behind-our-digital-infrastructure/)
+[](http://ford)[elsewhere](https://numfocus.org/programs/sustainability)
```
I think this empty link is a typo:
```suggestion
[elsewhere](https://numfocus.org/programs/sustainability)
```

QuLogic commented on content/posts/matplotlib-rsef/index.md:
```diff
@@ -0,0 +1,51 @@
+---
+
+
+We are happy to announce that we have hired Elliott Sales de Andrade ([qulogic](https://github.com/qulogic))
```
```suggestion
We are happy to announce that we have hired Elliott Sales de Andrade ([QuLogic](https://github.com/QuLogic))
```

tacaswell replied:
Sorry about that.

tacaswell committed:
DOC: fix capitalization and empty link

Co-Authored-By: Elliott Sales de Andrade <quantum.analyst@gmail.com>

QuLogic reviewed (approved):
Don't forget to un-draft it when publishing (and change the date if you wish.)

story645 committed:
reorganized things, 

trying to give big picture before details + explicit ciitations instead of burying them in links.

tacaswell committed:
DOC: make not draft

tacaswell commented:
I flipped to not-draft, leaving to @QuLogic to push the button as final approval.

QuLogic commented:
I don't have that power on this repo, but it LGTM.
